cli/Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
 name = "phylum-cli"
 version = "3.5.0"
-authors = ["Eric Freitag <eric@phylum.io>"]
-edition = "2018"
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
+edition = "2021"
+rust-version = "1.60"
```

Comment:
Does this change warrant a reminder to devs to rustup to this version?

Reply:
The hope of adding it to Cargo.toml is that devs will get a helpful error message if they have not updated